src/agent/tools/assess_retrieval_confidence.py

```
@@ -0,0 +1,208 @@
+    - Use AFTER a search (requires chunk IDs from search results)
+    - High confidence (≥0.85): Safe to use for automated response
+    - Medium confidence (0.70-0.84): Review recommended
+    - Low confidence (<0.70): Mandatory review or alternative retrieval
+
+    **Returns:** Confidence score (0-1) with detailed breakdown
+
+    **Method:** Score analysis from hybrid search metadata
+    
+    """
+
+    input_schema = AssessRetrievalConfidenceInput
```

Ruff (0.14.5)

Undefined name `AssessRetrievalConfidenceInput`

(F821)